proc_stat_test.go

```diff
@@ -38,7 +38,7 @@ func TestProcStat(t *testing.T) {
 		{name: "user time", want: 1677, have: int(s.UTime)},
 		{name: "system time", want: 44, have: int(s.STime)},
 		{name: "start time", want: 82375, have: int(s.Starttime)},
-		{name: "virtual memory size", want: 56274944, have: s.VSize},
+		{name: "virtual memory size", want: 56274944, have: int(s.VSize)},
```

Comment:
You should cast the literal here. `uint(56274944)`.

Reply:
sure, I can change this... just did the same as for the other cases above....

Reply:
Using uint here makes the test compilation fail...
Removing the last commit again...

Reply:
Hrm, My Go testing fu is failing I guess.